src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -153,6 +159,19 @@ public HostOutputMessage HostOutputMessage
             return exceptionInfo != null ? new ExceptionDispatchInfoDelayedException(exceptionInfo) : null;
         }
 
+        private async Task InvokeExceptionFiltersAsync(Exception exception, IFunctionInstance functionInstance, CancellationToken cancellationToken)
+        {
+            var exceptionFilters = GetFilters<IFunctionExceptionFilter>(_extensionRegistry, functionInstance.FunctionDescriptor, functionInstance.JobInstance);
+            if (exceptionFilters.Any())
+            {
+                var exceptionContext = new FunctionExceptionContext(functionInstance.Id, functionInstance.FunctionDescriptor.ShortName, _logger, exception);
+                foreach (var exceptionFilter in exceptionFilters)
+                {
+                    await exceptionFilter.OnExceptionAsync(exceptionContext, cancellationToken);
```

Comment:
>OnExceptionAsync [](start = 42, length = 16)

This is user code - what should we do if it throws?   
1. Keep executing other eh filters?
2. Does that become the new FunctionResult? 
Throwing from an EH filter probably ought to be similar to throwing from a Post filter.